src/commands/Minion/slayertask.ts

```diff
@@ -32,13 +34,42 @@ export default class extends BotCommand {
 		});
 	}
 
+	public getAlternateMonsterList(currentTask: SlayerTaskTable, assignedTask: AssignableSlayerTask | null) {
+		if (currentTask && assignedTask) {
+			const altMobs = assignedTask.monsters;
+			const alternateMonsters = killableMonsters
+				.filter(m => {
+					return altMobs.includes(m.id) && m!.id !== assignedTask.monster.id;
+				})
+				.map(m => {
+					return m!.name;
+				});
+			return alternateMonsters.length > 0 ? ` (${alternateMonsters.join('/')})` : '';
+		}
+		return '';
+	}
+
+	public async returnSuccess(msg: KlasaMessage, message: string, autoslay: boolean) {
+		if (autoslay) {
+			// await msg.channel.send(message);
+			try {
+				return this.client.commands.get('autoslay')!.run(msg, ['']);
+			} catch (e) {
+				return msg.channel.send('It was not possible to auto-slay this task. Please, try again.');
+			}
+		}
+		return msg.channel.send(message);
+	}
+
 	@requiresMinion
 	async run(msg: KlasaMessage, [input]: [string | undefined]) {
 		const { currentTask, totalTasksDone, assignedTask } = await getUsersCurrentSlayerInfo(msg.author.id);
 		const myBlockList = msg.author.settings.get(UserSettings.Slayer.BlockedTasks);
 		const myQPs = msg.author.settings.get(UserSettings.QP);
-
 		const maxBlocks = calcMaxBlockedTasks(myQPs);
+
+		let returnMessage = '';
+
```

Comment:
Maybe you could also add the attackable monsters' names to the 'busy' message: "Your minion is busy, but you can still manage your block list: +st blocks
You have 276 slayer points, and have completed 38 tasks in a row. XXX out of YYY MONSTER_NAME (Alternates) remaining"

Reply:
Don't think this is necessary.